lib/generators/paper_trail/install/templates/create_versions.rb.erb

```diff
@@ -28,7 +28,9 @@ class CreateVersions < ActiveRecord::Migration<%= migration_version %>
       # MySQL users should also upgrade to at least rails 4.2, which is the first
       # version of ActiveRecord with support for fractional seconds in MySQL.
       # (https://github.com/rails/rails/pull/14359)
-      #
+      # 
+      # MySQL users should use the following line for `created_at`
+      # t.datetime :created_at, limit: 6                                        
```

Comment:
Hi @jaredbeck, do you use MySQL to verify this?

Reply:
`limit` is the correct option, IIRC. I don't remember what the limit should be though. I think `6` will give microseconds precision. That's probably fine. In rare cases, some apps may choose a different number, but it's probably fine if we recommend `6`.

> Hi @jaredbeck, do you use MySQL to verify this?

Yes, the CI runs an actual MySQL instance. You can also run MySQL locally if you want.